workerapi/worker.thrift

```diff
@@ -0,0 +1,38 @@
+enum Status {
+  UNKNOWN = 0
+  READY = 1
+  PENDING = 2
+  FAILURE = 3
```

Comment:
Failure and Success seem like the same to the worker. Maybe collapse them into COMPLETED?